packages/ember-metal/lib/mixin.js

```diff
@@ -362,6 +389,18 @@ function replaceObserversAndListeners(obj, key, observerOrListener) {
   }
 }
 
+// function elemEq(arr1, arr2) {
```

Comment:
this was for the assert while I was developing to make sure the Object.keys(values) matched keys